java/src/main/java/vowpalWabbit/learner/VWLearners.java

```diff
@@ -1,8 +1,6 @@
 package vowpalWabbit.learner;
 
-import vowpalWabbit.jni.NativeUtils;
-
-import java.io.IOException;
+import java.util.Optional;
```

Comment:
This won't work.  The `pom.xml` file specifies the language level as Java 6 and this is a Java 8 construct.